app/src/main/java/com/wizeline/mobilenews/ui/dashboard/ArticleItem.kt

```diff
@@ -11,6 +11,7 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.blur
 import androidx.compose.ui.graphics.Brush
```

Comment:
There are some imports in the file that are not used any more. Just delete them with `ctrl` + `option` + `o` and you're good to go.